src/layout/StepsAccordion.js

```diff
@@ -0,0 +1,112 @@
+import _ from 'lodash/fp'
+import F from 'futil-js'
+import { observer } from 'mobx-react'
+import React from 'react'
+import DefaultIcon from '../DefaultIcon'
+import { Flex } from './Flex'
+import { splitKeys } from '../utils/futil'
+
+// Observes node, so we can activate the Continue button if it (or any child) has a value.
+// We don't observe on Step because then it would rerender its children when `node`
+// changes, which unfocuses query inputs as soon as the first character is entered.
+let Buttons = observer(
+  ({ step, totalSteps, currentStep, Button, Icon, onSubmit }) => (
+    <>
+      {step > 0 && (
+        <Button onClick={F.sets(step - 1, currentStep)} className="back-button">
+          <Icon icon="PreviousPage" />
+          Back
+        </Button>
+      )}
+      {step < totalSteps - 1 ? (
+        <Button
+          primary
+          onClick={F.sets(step + 1, currentStep)}
+          disabled={false}
+        >
+          Continue
+        </Button>
+      ) : (
+        <Button primary onClick={onSubmit}>
+          View Results
+        </Button>
+      )}
+    </>
+  )
+)
+
+let Step = ({
+  Button,
+  Icon,
+  style,
+  className,
+  step,
+  totalSteps,
+  currentStep,
+  stepTitle = '',
+  isRequired = false,
+  onSubmit,
+  children,
+}) => {
+  let isOpen = F.view(currentStep) === step
+  return (
+    <div className={`wizard-step ${className ? className : ''}`} style={style}>
+      <Flex alignItems="center" justifyContent="space-between">
+        <Flex alignItems="center">
+          <h1>
+            <span className="step-number">Step {step + 1}</span>
```

Comment:
Just a thought - it occurs to me that this might not always be desirable, so maybe it shouldn't be the responsibility of this component to add `Step n`. I don't feel super strongly about this though so I'm open - just wanted to bring it up